modules/virt-about-vtpm-devices.adoc

```diff
@@ -12,7 +12,28 @@ You can use a vTPM device with any operating system, but Windows 11 requires
 the presence of a TPM chip to install or boot. A vTPM device allows VMs created
 from a Windows 11 image to function without a physical TPM chip.
 
-A vTPM device also protects virtual machines by storing secrets without physical hardware. {VirtProductName} supports persisting vTPM device state by using Persistent Volume Claims (PVCs) for VMs. You must specify the storage class to be used by the PVC by setting the `vmStateStorageClass` attribute in the `HyperConverged` custom resource (CR):
+{VirtProductName} supports persisting vTPM device state by using Persistent Volume Claims (PVCs) for VMs. If you do not specify the storage class for this PVC, {VirtProductName} uses the default storage class for virtualization workloads. If the default storage class for virtualization workloads is not set, {VirtProductName} uses the default storage class for the cluster.
+
+[NOTE]
+====
+The storage class that is marked as default for virtualization workloads has the annotation `storageclass.kubevirt.io/is-default-virt-class` set to "true". You can find this storage class by running the following command:
+
+[source,terminal]
+----
+$ oc get sc -o json | jq '.items[].metadata|select(.annotations."storageclass.kubevirt.io/is-default-virt-class"=="true")|.name'
+----
+
+Similarly, the default storage class for the cluster has the annotation `storageclass.kubernetes.io/is-default-class` set to "true". To find this storage class, run:
+
+[source,terminal]
+----
+$ oc get sc -o json | jq '.items[].metadata|select(.annotations."storageclass.kubernetes.io/is-default-class"=="true")|.name'
```

Comment:
Per the doc guidelines, [Do not use jq in commands (unless it is truly required)](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#commands-with-jq).